app/assets/stylesheets/course/groups.css.scss

```diff
@@ -0,0 +1,8 @@
+.course-groups {
+  &.new, &.edit{
```

Comment:
Each selector in a comma sequence should be on its own line<br>Opening curly brace `{` should be preceded by one space